### PR TITLE
Skeleton styling

### PR DESCRIPTION
**Pull request by Melisa-Zorraindo**
Mobile first approach

The settings menu feature dark/light themes and the possibility to increase, decrease, and reset the font size. The code to change font size looks duplicated because I had to create a new container to apply the different classes the plugin uses.

The audio settings menu could do with some icons to match the styling of the other settings menu, ideas are welcome!

I've overriden the default width TW typography plugin offers, it looked too narrow in my opinion. If you think the content is hard to read, I'll remove the new width.

All cards have the same look and feel.

### CODE REVIEW

File: packages/lms/src/routes/search/+page.svelte [hunk 1]
```diff
@@ -129,243 +125,181 @@
...
+						<header class="card-header border-b flex justify-between items-baseline gap-3 p-0">
+							<h3 class="h3">{result.title}</h3>
+							<span class="chip variant-filled">{result.type}</span>
+						</header>
+						<footer class="card-footer p-0 pt-2">
+							{#each result.tags as tag}
+								<p class="chip variant-ringed m-1 ms-0 pt-2">{tag}</p>
+							{/each}
+						</footer>
+					</a>
```
Comment (Anclagen) [inline packages/lms/src/routes/search/+page.svelte:300]: Perhaps we should make a component for this as it needs to be reused on the tags page. Which currently is lacking styles on its cards now.

File: packages/lms/src/app.html [hunk 2]
```diff
@@ -12,6 +12,8 @@
 		%sveltekit.head%
 	</head>
 	<body data-sveltekit-preload-data="hover">
-		<div style="display: contents">%sveltekit.body%</div>
+		<div style="display: contents" class="h-full overflow-hidden text-base" id="container" data-font-size-index="1">
```
[the remark this answers is not in the document]
Comment (Melisa-Zorraindo) [reply packages/lms/src/app.html:15]: I followed the official docs https://www.skeleton.dev/components/app-shell#prerequisites. Did I get it wrong?

[the remark this answers is not in the document]
Comment (pretzL) [reply packages/lms/src/app.html:15]: Nah, leave it as-is then. There should be a reason as to why this is done this way when Tailwind supports the `display: contents` natively.

**Review by pretzL — APPROVED** [review]

File: packages/lms/src/routes/search/+page.svelte (hunk 1 above)
Comment (Melisa-Zorraindo) [reply packages/lms/src/routes/search/+page.svelte]: Added

**Review by Jenfer93 — APPROVED** [review]

**Pull request merged**